tensorflow/core/kernels/cwise_ops_test.cc

```diff
@@ -25,9 +25,9 @@ namespace tensorflow {
 
 // Creates a Graph which applies a unary "func" on a 3D float tensor
 // of "num" elements.
-static Graph* Unary(const string& func, int num) {
+static Graph* Unary(const string& func, int num, DataType dtype) {
   Graph* g = new Graph(OpRegistry::Global());
-  Tensor data(DT_FLOAT, TensorShape({64, 64, num / (64 * 64)}));
+  Tensor data(dtype, TensorShape({64, 64, num / (64 * 64)}));
   CHECK_GT(data.NumElements(), 0);
   data.flat<float>().setRandom();
```

Comment:
Hm this actually crashes at runtime if you run the benchmark (Jenkins doesn't) since .flat<float> doesn't match the type when it's complex<float> or complex<double>.